fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/CardNudge.kt

```diff
@@ -130,9 +134,15 @@ fun CardNudge(
         ) {
             LaunchedEffect(state.offset.value) {
                 if (state.offset.value > 0.1F) {
-                    metadata.rightSwipeGesture?.invoke(state.offset.value / maxWidth)
+                    when {
+                        isRtl -> metadata.rightSwipeGesture?.invoke(-state.offset.value / maxWidth) //negative value for RTL for aligning the "Hide" Text appropriately
+                        else -> metadata.rightSwipeGesture?.invoke(state.offset.value / maxWidth)
```

Comment:
Revert this change. It is consumer responsibility to process it as per layout direction.

Reply:
Done,
also  fixed this: (Now it shows "Left Swiped: some value"
![Screenshot_2023-08-30-15-23-13-45_138c6ebfedc8e8a8cd1938476bab217a](https://github.com/microsoft/fluentui-android/assets/68989156/3af033a1-2cd0-4484-b95e-f7d227a06169)